### PR TITLE
README:fix and update

### PR DESCRIPTION
**Pull request by andy5995**
#3 and #5 [skip ci]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,39 @@
 # energy-calculator
-calculates the energy of different objects based on the equation E=mc2
+Educational tool and calculator for energy-based concepts
```
Comment (andy5995) [inline README.md:2]: Any suggestions for the description, @Jammyjamjamman ?

**Comment by andy5995** [comment]
@Jammyjamjamman Now that you've made the change so that EC can be used as a library, we should probably make a note of that on the README.

File: README.md (hunk 1 above)
Comment (Jammyjamjamman) [reply README.md:2]: Perhaps
```
Educational tool for performing physics calculations.
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
```
Comment (andy5995) [inline README.md:2]:
```suggestion
Educational tool for performing physics calculations.
```

**Pull request merged**